deploy/crds/mongodb.com_mongodb_crd.yaml

```diff
@@ -169,10 +250,10 @@ spec:
                       type: object
                     type: array
                   scramCredentialsSecretName:
-                    description: ScramCredentialsSecretName appended by string "scram-credentials" is the name of the secret object
-                      created by the mongoDB operator for storing SCRAM credentials
+                    description: ScramCredentialsSecretName appended by string "scram-credentials"
+                      is the name of the secret object created by the mongoDB operator
+                      for storing SCRAM credentials
                     type: string
-                    pattern: ^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$
```

Comment:
Should I add this back? The regex was not accepted by the auto-generation tool, so I commented it out.

Reply:
What is the error that we get from it? 

If it's an easy fix I think we can just do it here, if not maybe we can temporarily remove it and create a ticket to fix. Maybe @irajdeep knows a bit more about this?

Reply:
let's leave it for now and we can address this in a future PR.

Reply:
Done!